.github/workflows/build_and_deploy.yml

```
@@ -374,6 +375,22 @@ jobs:
         if: ${{ !matrix.settings.docker }}
         shell: bash
 
+      # Trying to upload metrics for the Turbopack to datadog's CI pipeline execution
+      - name: 'Upload turbopack build metrics'
+        shell: bash
+        continue-on-error: true
```

This should only run the same as `upload artifact` right or are we wanting to report duplicate metrics even when a cache hit occurs? 
```suggestion
if: ${{ needs.build.outputs.isRelease == 'true' }}
```

---

I originally wanted to record cache hit too to record cache hit ratio. Maybe it's not needed?

---

Updated. `continue-on-error` still remains just in case.